docs/get-started/mapbox-tokens.md

```diff
@@ -1,20 +1,28 @@
 # About Mapbox Tokens
 
-react-map-gl and the underlying Mapbox GL JS libraries are open source and free to use. However, to load the map styles and tiles from Mapbox's data service, you will need to register on their website in order to retrieve an access token required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a certain level of traffic is exceeded.
+react-map-gl is open source and free. The underlying Mapbox GL JS library was free to use up to version 1.x, only requiring an access token if you load the map styles and tiles from Mapbox's data service. As of [v2.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0), Mapbox GL JS must be used with an access token in order to access the map renderer.
```

Comment:
```suggestion
`react-map-gl` itself is open source and free. It is designed to provide React components for mapbox-gl-js or derived projects. 

Depending on which mabpox-gl-js fork you use, you may need a mapbox token. You will need a mapbox token if you use:
-  [Mapbox GL JS v2.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v2.0.0) requires a mapbox access token in order to access the map renderer, and generates billable events regardlesss of whether you are displaying your own maps.
-  [Mapbox GL JS v1.0](https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.0.0) requires an access token only if you load the map styles and tiles from Mapbox's data service. If you display non-mapbox maps, you do not need a mapbox token.
```

Reply:
`mabpox`

Reply:
The module is called `mapbox-gl`. The official name is Mapbox GL JS.

Reply:
I was trying to point out the `mabpox` typo in

> Depending on which mabpox-gl-js fork you use ...